.github/workflows/actions.yml

```diff
@@ -83,22 +90,23 @@ jobs:
             client/dist/mac-portable.zip
   build-universal2:
     name: Build NSO-RPC - Universal2
-    runs-on: macos-12
+    runs-on: macos-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
-        with:
-          python-version: 3.11.4
       - name: Install Python 3.11.4 and build NSO-RPC
```

Comment:
May be a good idea to remove "and build NSO-RPC" from this, as you have a dedicated build step below.